queries/fennel/rainbow-delimiters.scm

```diff
@@ -1,31 +1,31 @@
 (list
-  "(" @opening
-  ")" @closing) @container
+  "(" @delimiter
+  ")" @delimiter @sentinel) @container
 
 (table
-  "{" @opening
-  (":" @intermediate)*
-  "}" @closing) @container
+  "{" @delimiter
+  (":" @delimiter)*  ; FIXME: Only the first one is highlighted
```

Comment:
```suggestion
  (":" @delimiter _)*  ; FIXME: Only the first one is highlighted
```
This seems to fix it for me, but I am not sure if we generally know that the type of `_` is `symbol` (that is what it is in the example code). If so, replace `_` with `(symbol)`.

Reply:
Thanks, that's a great trick. It will come in handy for other languages as well.